<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>141</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    alternate-background-color: blue;
    background: black;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QPushButton" name="bt_neural_scheme">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="scheme/graphics/graphics.qrc">
       <normaloff>:/graphics/schemeNeurons.jpg</normaloff>:/graphics/schemeNeurons.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>121</width>
       <height>91</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_general_monitor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>121</width>
       <height>91</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="scheme/graphics/graphics.qrc">
       <normaloff>:/graphics/generalMonitorMain.png</normaloff>:/graphics/generalMonitorMain.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>121</width>
       <height>91</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_exit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>440</y>
       <width>121</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="scheme/graphics/graphics.qrc">
       <normaloff>:/graphics/exit.png</normaloff>:/graphics/exit.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>71</width>
       <height>71</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_about">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>297</y>
       <width>121</width>
       <height>91</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="scheme/graphics/graphics.qrc">
       <normaloff>:/graphics/about3.jpg</normaloff>:/graphics/about3.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>121</width>
       <height>91</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_neural_scheme">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
background: transparent;

</string>
     </property>
     <property name="text">
      <string>Neural Scheme</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_general_monitor">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>254</y>
       <width>118</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
background: transparent;

</string>
     </property>
     <property name="text">
      <string>General Monitor</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lb_about">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
background: transparent;

</string>
     </property>
     <property name="text">
      <string>About us</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lb_name">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>520</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>NESIM-RT</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_title">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>781</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>A Real Time Spiking Neural Networks Simulator</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>7</width>
      <height>611</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">    alternate-background-color: blue;
    background: green;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lb_logo">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>500</y>
      <width>76</width>
      <height>102</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="scheme/graphics/graphics.qrc">:/graphics/logoUCA1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_background">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>781</width>
      <height>431</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="scheme/graphics/graphics.qrc">:/graphics/neuron_background.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="scheme/graphics/graphics.qrc"/>
  <include location="scheme/graphics/graphics.qrc"/>
 </resources>
 <connections/>
</ui>
